processes/security_team_onboarding.md

```diff
@@ -0,0 +1,39 @@
+# On-boarding New Security WG Members
+
+This document aims to provide a clear process for on-boarding new team members to the Node.js Security WG.
+
+## Introducing new members
+New members may self-nominate themselves, or be nominated to join the WG by existing members. They share an affinity to web or software security research, or activism around security and the Node.js project. All of which make good criteria to join the team but are not a closed list.
+
+### The Buddy Program
+Every new member should be assigned a *buddy* to help mentor and coach them through the on-boarding period, relevant processes and activities that the WG handles. 
+
+A *buddy* is an active member of the WG who is familiar with the above topics, and should be recommended on the nomination issue/PR.
+
+A suggested on-boarding session for triage team members: 30 minutes touch-base call to cover the triage process on HackerOne and the repo.
```

Comment:
This might be specific to people who will be involved in the triage of 3rd party vulnerabilities which is not necessarily everybody in the security-wg.

Reply:
That's what I'm wondering - even if you're not in the triage team, wouldn't you want a quick tour of stuff we do, people involved, processes, etc? or would you say that all of that can be retrieved from the repo and our documents?